tools/travis/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+docker_image_prefix="testing"
+dockerhub_image_prefix="openwhisk"
+docker login -u "${DOCKER_USER}" -p "${DOCKER_PASS}"
+
+#capture couchdb setup
+docker commit couchdb "${docker_image_prefix}/couchdb"
+
+#deploy specific openwhisk core images
+for container in "controller"     \
+                 "invoker"        \
+                 "couchdb"        \
```

Comment:
Yes that's the idea to have a db with views already ready with and working out of the box for simple scenarios like local development deploying with docker-compose and a default subjects installed.

We could also have the catalog repo, pusing a couchdb_catalog, ready to go with system packages already pre-installed.